universe/Enums.h

```diff
@@ -147,6 +147,14 @@ GG_ENUM(Shape,
     GALAXY_SHAPES   ///< the number of shapes in this enum (leave this last)
 )
 
+/** levels of available technology at game start */
+GG_ENUM(StartingEra,
+    INVALID_STARTING_ERA = -1,
+    STARTING_ERA_PREWARP,
+    STARTING_ERA_ADVANCED,
```

Comment:
probably rename as default or something meaningful, and make the default with value 0